### PR TITLE
[docs][caps] Updates for Astra Linux Parsec

### PR DESCRIPTION
**Pull request by borg-z**
## Description

Added instructions for configuring the `caps` user in Astra Linux SE to prevent privilege issues when using the Parsec module.

## Why do we need it, and what problem does it solve?

Fixes an issue where `staticinstances.deckhouse.io` fail to bootstrap due to loss of caps user privileges.

## What is the expected result?

Astra Linux SE Static Instance should boot and run successfully.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix 
summary: Added steps to configure integrity level for Astra Linux SE to prevent permission issues.
impact: 
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'deploy/web/stage'...

File: modules/040-node-manager/docs/EXAMPLES_RU.md [hunk 1]
```diff
@@ -146,7 +146,13 @@ spec:
...
      ```
 
-1. Создайте в кластере ресурс [SSHCredentials](cr.html#sshcredentials).
+   * В операционных системах семейства Astra Linux, при использовании модуля мандатного контроля целостности Parsec, сконфигурируйте максимальный уровень целостности для пользователя `caps`:
+
+     ```shell
+     pdpl-user -i 63 caps
+     ```
+
+2. Создайте в кластере ресурс [SSHCredentials](cr.html#sshcredentials).
```
Comment (z9r5) [inline modules/040-node-manager/docs/EXAMPLES_RU.md:155]:
```suggestion
1. Создайте в кластере ресурс [SSHCredentials](cr.html#sshcredentials).
```

File: modules/040-node-manager/docs/EXAMPLES_RU.md [hunk 2]
```diff
@@ -174,7 +180,7 @@ spec:
    EOF
    ```
 
-1. Создайте в кластере ресурс [StaticInstance](cr.html#staticinstance), указав IP-адрес сервера статического узла:
+3. Создайте в кластере ресурс [StaticInstance](cr.html#staticinstance), указав IP-адрес сервера статического узла:
```
Comment (z9r5) [inline modules/040-node-manager/docs/EXAMPLES_RU.md:183]:
```suggestion
1. Создайте в кластере ресурс [StaticInstance](cr.html#staticinstance), указав IP-адрес сервера статического узла:
```

File: modules/040-node-manager/docs/EXAMPLES_RU.md [hunk 3]
```diff
@@ -191,7 +197,7 @@ spec:
    EOF
    ```
 
-1. Создайте в кластере ресурс [NodeGroup](cr.html#nodegroup):
+4. Создайте в кластере ресурс [NodeGroup](cr.html#nodegroup):
```
Comment (z9r5) [inline modules/040-node-manager/docs/EXAMPLES_RU.md:200]:
```suggestion
1. Создайте в кластере ресурс [NodeGroup](cr.html#nodegroup):
```

File: modules/040-node-manager/docs/EXAMPLES_RU.md [hunk 4]
```diff
@@ -146,7 +146,13 @@ spec:
      chown -R caps:caps /home/caps/
      ```
 
-1. Создайте в кластере ресурс [SSHCredentials](cr.html#sshcredentials).
+   * В операционных системах семейства Astra Linux, при использовании модуля мандатного контроля целостности Parsec, сконфигурируйте максимальный уровень целостности для пользователя `caps`:
```
Comment (z9r5) [inline modules/040-node-manager/docs/EXAMPLES_RU.md:149]:
```suggestion

   В операционных системах семейства Astra Linux, при использовании модуля мандатного контроля целостности Parsec, сконфигурируйте максимальный уровень целостности для пользователя `caps`:
```

**Review by name212 — APPROVED** [review]

**Pull request merged**